### DISCUSSION TITLE
multiple output directories

### DISCUSSION THREAD

**Discussion by rputikar** [quarto-dev/quarto-web#773] [Q&A]
Is it possible to have multiple output directories in _quarto.yml based on the format type?

e.g.
format: 
  revealjs:
    output-dir: ./slides
  pdf:
    output-dir:./pdfs

**Comment by cderv** [comment]
`output-dir` is a project configuration. For now, you would need to use [Project Profile](https://quarto.org/docs/projects/profiles.html) if you want to change configuration for a project depending on how you render. 

You could have two profiles : One for revealjs, and one for Pdf and then activating the profile required when rendering
